manifests/e/EaseUS/EaseUSTodoPCTrans/13.2/EaseUS.EaseUSTodoPCTrans.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: EaseUS.EaseUSTodoPCTrans
+PackageVersion: 13.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://download3.easeus.com/free/pctrans.exe
+  InstallerSha256: 01785CF26EBB27C2EE2B0861359355100A3278400523BC0E8281E2BBF778B20B
+  InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress

```

Comment:
As you asked in the discussion - if all are present, field isn't required, but its not a bad idea to include anyways. Even though the cli doesn't do anything with this information currently, it likely will in the future

Reply:
Ok. Thanks for the information!